python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -360,11 +400,12 @@ def __contextMenu( column, pathListing, menuDefinition ) :
 			"command" : functools.partial( __editSelectedCells, pathListing, toggleOnly ),
 		}
 	)
+	tweaks, tweaksEnabled = __toggleableInspectionTweaks( pathListing )
 	menuDefinition.append(
-		"Disable Edit",
+		"{} Edit{}".format( "Disable" if tweaksEnabled else "Enable", "s" if len( tweaks ) > 1 else "" ),
```

Comment:
Perhaps "Reenable" rather than "Enable"? Maybe that suggests something that was recently disabled a little better?

Reply:
Switched to "Reenable" in 8368682.